src/Sulu/Bundle/HttpCacheBundle/Tests/Unit/DependencyInjection/SuluHttpCacheExtensionTest.php

```diff
@@ -22,6 +22,8 @@ public function setUp()
         $this->container->setParameter('kernel.environment', 'test');
         $this->container->set('sulu_core.webspace.webspace_manager', $this->getMock('Sulu\Component\Webspace\Manager\WebspaceManagerInterface'));
         $this->container->set('sulu.content.type_manager', $this->getMock('Sulu\Component\Content\ContentTypeManagerInterface'));
+        $this->container->set('request_stack', $this->getMock('Symfony\Component\HttpFoundation\RequestStack'));
+        $this->container->set('sulu_core.webspace.webspace_manager.url_replacer', $this->getMock('Sulu\Component\Webspace\Url\Replacer'));
```

Comment:
i think the `container`-variable is initialized in the `setUp`-method `AbstractContainerBuilderTestCase`-class